CHANGELOG.md

```diff
@@ -22,6 +22,23 @@
 - **Breaking:** On Web, `set_cursor_position` and `set_cursor_grab` will now always return an error.
 - **Breaking:** `PixelDelta` scroll events now return a `PhysicalPosition`.
 - On NetBSD, fixed crash due to incorrect detection of the main thread.
+- On Wayland, add missing virtual keycodes.
+- On Wayland, implement proper `set_cursor_grab`.
+- On Wayland, the cursor will use similar icons if the requested one isn't available.
+- On Wayland, right clicking on decorations could show application menu.
```

Comment:
This sounds a bit funny. Maybe better to phrase it more like `show application menu when right clicking on decorations if the compositor supports it`. Though that might be a tad long.

Reply:
I decided to go with `will request` instead of `could show`. Should be pretty clear that it's not guaranteed to be shown.